commands/login.go

```diff
@@ -145,7 +145,7 @@ func (command *LoginCommand) authCodeGrant(targetUrl string) (string, string, er
 		panic(err)
 	}
 
-	openURL := fmt.Sprintf("%s/sky/login?redirect_uri=%s\n", targetUrl, redirectUri.String())
+	openURL := fmt.Sprintf("%s/sky/login?redirect_uri=%s", targetUrl, redirectUri.String())
 
 	fmt.Println("navigate to the following URL in your browser:")
 	fmt.Println("")
```

Comment:
Need a `\n` after the `%s` in the `fmt.Printf` below this line now to keep the formatting as it was before.